src/Gaffer/MonitorAlgo.cpp

```diff
@@ -318,6 +319,9 @@ struct FormatStatistics
 		}
 
 		std::stringstream s;
+
+		s << "Total " << metric.description() << " : " <<  metric( combinedStatistics ) << "\n";
```

Comment:
I'm finding this formatting really awkward to read. Could we put all the totals in a single section on their own, with the same indentation we have for the other sections?

```
PerformanceMonitor Summary :

  Time spent in hash and compute processes : x.xx
  Time spent in hash processes             : x.xx
  Time spent in compute processes          : x.xx
  ...
```